tools/tests/com.liferay.ide.project.core.tests/src/com/liferay/ide/project/core/tests/MavenLegacyPomUpdaterTests.java

```diff
@@ -0,0 +1,189 @@
+/*******************************************************************************
```

Comment:
can I just make an interface in project.core and make a implementation as a osgi component in maven.core and register it and the call it in project core ? I think it would be simpler